rs/nns/governance/src/storage.rs

```diff
@@ -122,6 +122,7 @@ impl State {
     fn validate(&self) {
         self.stable_neuron_store.validate();
         self.stable_neuron_indexes.validate();
+        validate_stable_log(&self.audit_events_log);
```

Comment:
The title says np rewards but the code says audit event logs. Just a reminder to fix this before merging.